meetings/2020-12-15.rst

```diff
@@ -0,0 +1,76 @@
+    e.g., a few or a single way to represent a crystal structure. 
+    Depending on the goal with an ontology it may not be designed with the same desire, 
+    but rather one may aim to define many related concepts and alternatives for representation.
+    
+    - We may want to consider where our effort is positioned in this. To standardize for integration between databases, we may want to be restrictive, 
+      and accommodate the very wide range of alternative representations primarily via the definition of transformations to/from the ontology from other external ontologies (e.g., database-specific ontologies).
+
+  * CIF has a concept of "composite material" that is different from ours; a composite of overlapping repeating lattices.
+    We may want to consider adding this model and carefully consider naming in relation to our "Composite materials model."
+
+- Discussion about the division in "homogenous" and "inhomogenous" materials models
+
+  * Do we want to build this type of classifications into the "main tree" hierarchy, or not? 
+  
+    - Is there a point to adding extra steps in the "hierarchy of models"? Or, is there a desirable simplicity in keeping all specific materials models as the direct sublevel below "Materials model"? 
+      
+    - We can still have 'separate' classification concepts (e.g., entities) that we tie to the models, e.g., homogenous and inhomogenous, that would allow ontology-based reasoning on these concepts.
+
+- A similar question about hierarchical representation is about the "single structure" materials model we presently have in the prototype ontology.
+  This naming suggests a concept that encompasses many "submodels" that all fall under 'one structure.'
+  
+  * This may mostly be a naming thing in the prototype ontology. Our description relates more closely to "Single crystal," so perhaps we just need to rename "Single structure materials model" -> "Single crystal materials model."
+  
+  * We may want to branch out a few other models: Powder models? (Single material powder? Multiple materials powder?) Monolithed structures.
+   
+- Future / Agenda for the next meeting
+
+  - Consider renaming "Single structure materials model" -> "Single crystal materials model." However, this may need more discussion;
+    what more exactly is "a structure" for us?
+  
+  - Consider adding more models of the kinds discussed above.
+
+  - We have now had a series of explorative discussions of designs and concepts. 
+    Next meeting, we may want to try to move over to more concretely discuss how to progress with the prototype ontology in terms of
+    new entities (atoms, states, properties, ...) and materials model representations, e.g., pin down what "single structure" materials models 
+    means in a larger degree of precision and maybe discuss representations of bulk materials with defects.
+
+  - Next meeting will be at the end of January. Rickard sends a doodle.
```

Comment:
This change suggestion inserts useful links provided by @jamesrhester in the comment part of the PR
```suggestion
  - Next meeting will be at the end of January. Rickard sends a doodle.
  
Supplementary notes
-------------------
* Useful links related to the CIF standard:

  - `Standards development <https://github.com/COMCIFS>`__

  - `dREL in particular <https://github.com/COMCIFS/dREL>`__

  - `The core dictionary <https://github.com/COMCIFS/cif_core/blob/master/cif_core.dic>`__

  - `The DDLm language defined using itself <https://github.com/COMCIFS/cif_core/blob/master/ddl.dic>`__

  - `The official IUCr website for CIF standards <https://www.iucr.org/resources/cif>`__ - the dictionaries are in an older DDL but the general architecture is the same.

  - `The 2005 International Tables For Crystallography Vol G <https://it.iucr.org/G>`__ - everything to do with CIF including software, dictionaries, core standards but doesn't include DDLm or CIF2 syntax.

```